### ISSUE TITLE
TrueType components more than 1 level deep are not handled by getGlyphSVGpath()

### ISSUE THREAD

**Issue by Lorp** [Lorp/samsa#13]
Example, RobotoExtremo-VF.ttf:

```
    <TTGlyph name="etilde" xMin="85" yMin="-30" xMax="1008" yMax="1446">
      <component glyphName="e" x="0" y="0" flags="0x204"/>
      <component glyphName="tildecomb" x="32" y="-11" flags="0x4"/>
    </TTGlyph>
   .
   .
   .
    <TTGlyph name="tildecomb" xMin="156" yMin="1190" xMax="911" yMax="1457">
      <component glyphName="tilde" x="0" y="0" flags="0x204"/>
    </TTGlyph>
```

I was working on the assumption that macOS still did not handle such fonts. However, that bug seems to be fixed, so we must fix it in Samsa.

**Comment by Omnibus-Type** [comment]
Hi Laurence, We indicate some improvements that could be useful in Samsa:

• Words cannot be composed. In axis opsz it would be very useful.
• For the Glyphs palette: the panel is too long. Glyphs with components could be grouped.

**Comment by Lorp** [comment]
Fixed in ed207a6be04026bf4af1167827c92c03db22945d

`getGlyphSVGpath()` has been replaced by the .svgPath() method of SamsaGlyph, which only takes decomposed glyphs

@Omnibus-Type : could you please post these as separate issues?
